setup.py

```diff
@@ -32,6 +32,7 @@ def run_tests(self):
     'pytest-django',
     'pytest-pythonpath',
     'pytest-cov',
+    'django-reversion',
```

Comment:
Why Django-Reverse ? I never use it and there's link to that in Django-Money.

Reply:
Its a test requirement, does not install when you normally pip install. Just if you install in test mode

Reply:
If you follow the link above you will see that some tests rely on django-reversion to work: https://github.com/django-money/django-money/search?q=reversion&unscoped_q=reversion

When i first ran the tests, before adding any code they broke because of that non declared dependency. So i added it to the `test_requirements` so the next developers that might contribute to this project does not neet to figure it out :)

But that`s nothing to worry about, neither of these packages are installed with a regular pip install:

```
test_requirements = [
    'mock',
    'pytest>=3.1.0',
    'pytest-django',
    'pytest-pythonpath',
    'pytest-cov',
    'django-reversion',
]
```
To install them you need to run `pip install -e .[tests]` after cloning the repo to your local machine.